tests/functional/event_handler/test_api_gateway.py

```diff
@@ -660,7 +690,7 @@ def custom_method():
         ...
 
     # WHEN calling the handler
-    result = app({"path": "/some-call", "httpMethod": "OPTIONS"}, None)
+    result = app({"path": "/some-call", "httpMethod": "OPTIONS", "headers": {"Origin": "https://example.org"}}, None)
```

Comment:
nitpick: this is quite long now, perhaps a variable in the given section so it's easier to understand the intent of the test